pkg/api/types.go

```diff
@@ -1386,7 +1390,9 @@ func (p ClusterProfile) ClusterType() string {
 		return string(CloudGCP)
 	case
 		ClusterProfileIBMCloud,
-		ClusterProfileIBMCloudQE:
+		ClusterProfileIBMCloudQE,
+		ClusterProfileIBMCloudMultiPpc64le, // TODO: Check if the return value should be "ibmcloud-multi-ppc64le" instead of "ibmcloud"
```

Comment:
This is ultimately a billing question. Let's find out who owns the wallet for the IBM Cloud account(s)

Reply:
I suspect we will want "ibmcloud-multi-ppc64le" and "ibmcloud-multi-s390x" so that we can have our own cluster profiles. :)

Reply:
(which in turn will specify the leases).

Reply:
Thanks for the comments, updated :).